ga4gh/datamodel/ontologies.py

```diff
@@ -0,0 +1,82 @@
+"""
+Support for Ontologies.
+FIXME: this is currently hard-coded for the GENCODE set as a short-term hack
+to support BRCA changeling.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import ga4gh.protocol as protocol
+
+
+class OntologyTermSet(object):
```

Comment:
We should inherit from ``datamodel.DatamodelObject`` here, That way, we'll inherit a lot of useful functionality, particularly dealing with compound ids.